src/ray/raylet/lib/java/org_ray_runtime_raylet_RayletClientImpl.cc

```diff
@@ -56,7 +56,7 @@ JNIEXPORT jlong JNICALL Java_org_ray_runtime_raylet_RayletClientImpl_nativeInit(
 JNIEXPORT void JNICALL Java_org_ray_runtime_raylet_RayletClientImpl_nativeSubmitTask(
     JNIEnv *env, jclass, jlong client, jbyteArray cursorId, jobject taskBuff, jint pos,
     jint taskSize) {
-  auto conn = reinterpret_cast<LocalSchedulerConnection *>(client);
+  auto client = reinterpret_cast<RayletClient *>(client);
```

Comment:
I think this would fail to compile, because variable `client` is re-defined. maybe rename the second `client` to `raylet_client`.